test/SlowTests/Client/Attachments/BulkInsertAttachments.cs

```diff
@@ -72,7 +72,7 @@ public async Task StoreManyAttachments(Options options, int count, int size)
             }
         }
 
-        [RavenTheory(RavenTestCategory.BulkInsert)]
+        [RavenRetryTheory(RavenTestCategory.BulkInsert)]
```

Comment:
Doesn't this change make `AllTestsShouldUseRavenFactOrRavenTheoryAttributes` test to fail?